apps/argos_aggregation/lib/argos_aggregation/data_sources/bibliography.ex

```diff
@@ -261,6 +275,19 @@ defmodule ArgosAggregation.Bibliography do
       end)
     end
 
+    defp parse_url([]) do
+      []
+    end
+
+    defp parse_url(url) do
+      %{"url" => url["url"],
+        "type" => :website,
+        "label" => [%{
+          "lang" => NaturalLanguageDetector.get_language_key(url["desc"]),
+          "text" => url["desc"]
```

Comment:
Bin mir gerade nicht sicher, ob "desc" in 100% der Fälle gesetzt ist oder auch mal leer sein könnte. Vielleicht wäre für nicht vorhanden/leer ganz gut einen Fallback-Wert zu haben.